### PR TITLE
Adminheal clears all non-appearance mutations.

### LINKED ISSUE #43547: Adminheal does not clear you of mutations.
## Round ID:
105738

## Reproduction:
1. Inject various admin syringe mutations in yourself.
2. Adminheal yourself.
3. Mutations remain.

We regenerate limbs and organs, why not DNA too?

### PR DESCRIPTION
**Pull request by ghost**
Adminheal/revive clears all non-appearance mutations, and apparently humanizes monkeyized players (fixes #43547).

### CODE REVIEW

**Comment by improvedname** [comment]
Idk what about positive powers man

**Comment by Time-Green** [comment]
Why non-appearance?

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -526,6 +526,7 @@
 			for(var/S in mind.spell_list)
 				var/obj/effect/proc_holder/spell/spell = S
 				spell.updateButtonIcon()
+		src:clean_dna()
```
Comment (Time-Green) [inline code/modules/mob/living/living.dm]: No need to use src, just use clean_dna()
Also use .’s next time

File: code/modules/mob/living/living.dm [hunk 2]
```diff
@@ -572,6 +573,7 @@
 		mood.remove_temp_moods(admin_revive)
 	update_mobility()
 	stop_sound_channel(CHANNEL_HEARTBEAT)
+	src:clean_dna()
```
Comment (Time-Green) [inline code/modules/mob/living/living.dm]: Ditto

**Comment by JohnParsnip** [comment]
@Time-Green appearance mutations do not influence game as much as non-appearance ones

File: code/modules/mob/living/living.dm (hunk 1 above)
Comment (JohnParsnip) [reply code/modules/mob/living/living.dm]: @Time-Green src: fixed some errors so that's why i used it

Comment (Barhandar) [reply code/modules/mob/living/living.dm]: Using : is a recipe for runtimes and maintainers will ree at you for it.

**Comment by Time-Green** [comment]
That's kind of a bad definition as what to remove and what not to remove.
Hulk, telekinesis and space adapt are very powerful mutations
Having an antenna stick out of your head, not really.
Aheal is supposed to be good. Just remove all non beneficial mutations, if you're up to that.

Also this doesn't actually remove all non-appearance mutation, it just removes all mutations

**Pull request closed without merging**